test/unit/gateways/redsys_sha256_test.rb

```diff
@@ -0,0 +1,383 @@
+require 'test_helper'
+
+class RedsysTest < Test::Unit::TestCase
+  include CommStub
+
+  def setup
+    Base.mode = :test
+    @credentials = {
+      :login      => '091952713',
+      :secret_key => "qwertyasdf0123456789",
+      :terminal   => '1',
```

Comment:
So, I'm now realizing that I might have accidentally overwritten some changes I made to the unit tests. :disappointed: But yes, there should be a `signature_algorithm`.